tests/modeladmin/test_checks.py

```diff
@@ -918,6 +948,16 @@ class TestModelAdmin(ModelAdmin):
             'admin.E103'
         )
 
+    def test_not_correct_inline_field(self):
+        class TestModelAdmin(ModelAdmin):
+            inlines = (10, )
```

Comment:
No trailing whitespace after , (there was a past PR that removed all cases like that)